modules/tinymce/src/plugins/paste/main/ts/api/Settings.ts

```diff
@@ -46,19 +46,6 @@ const isPasteAsTextEnabled = (editor: Editor): boolean =>
 const getRetainStyleProps = (editor: Editor): string | undefined =>
```

Comment:
This is Word specific logic as well so it should also be removed. See https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/#options